code_challenges/adhoc/counter.py

```diff
@@ -0,0 +1,15 @@
+# create a counter that has an incriment , decremet, and value method (return current value)
+class Counter:
+    def __init__(self):
+        self._value = 0
+        pass
+
+    def value(self):
+        value = self._value
+        return value
```

Comment:
When you find yourself assigning a variable only to return it, you don't need the assignment.